bluesky_queueserver/manager/qserver_cli.py

```diff
@@ -624,10 +651,16 @@ def create_msg(params):
     elif command == "queue":
         if len(params) < 1:
             raise CommandParameterError(f"Request '{command}' must include at least one parameter")
-        supported_params = ("add", "get", "clear", "item", "start", "stop")
+        supported_params = ("add", "update", "replace", "get", "clear", "item", "start", "stop")
         if params[0] in supported_params:
             if params[0] == "add":
-                method, prms = msg_queue_add(params)
+                method, prms = msg_queue_add_update(params, cmd_opt="add")
+
+            elif params[0] == "update":
+                method, prms = msg_queue_add_update(params, cmd_opt="update")
+
+            elif params[0] == "replace":
+                method, prms = msg_queue_add_update(params, cmd_opt="replace")
```

Comment:
All the 3 cases above can be merged into one case (like the one below), such as:
```py
if params[0] in ("add", "update", "replace")
    method, prms = msg_queue_add_update(params, cmd_opt=params[0])
```